packages/venia-ui/lib/util/images.js

```diff
@@ -55,7 +60,7 @@ export const generateSrcset = (imageURL, type) => {
 
     const generateSrcsetUrl = generateUrl(imageURL, type);
 
-    return Object.values(imageWidths)
+    return Array.from(imageWidths.values())
```

Comment:
The second argument to `Array.from()` is a mapping function, which will let you optimize by avoiding the intermediate array.

Reply:
Nice point. I didn't know that. Done.